setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     dataclasses; python_version<"3.7"
     Werkzeug
     enlighten
+    types-pkg-resources
```

Comment:
AFAIK this is only needed for typing. The package you need to import `pkg_resources` is named `setuptools`.
But setuptools is pretty big to install and slow to import for what it does. For getting the version you can now use [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html) (or [importlib_metadata](https://pypi.org/project/importlib-metadata/) in older Python, as with dataclasses).

But IMO, all this is unnecessary. The version is `1`, no need to extract it dynamically. If the config format changes in an incompatible way, we'll need to change the code carefully anyway, to make sure older config still works.